test/setup.ts

```diff
@@ -18,14 +18,18 @@ before(async () => {
 
 	await dropAllTables(sql);
 	await sql.migrate.latest();
-	await sql.seed.run();
+
+	await sql.seed.run({
+		loadExtensions: [ '.js', '.ts' ],
```

Comment:
I see you first set it to just JS, then added TS, is it still needed? Both are enabled by default.

Reply:
Wallaby.js requires `.js` extension, but standard test runner requires `.ts`. I ended up specifying both, so none of them throws.